vimeo-networking/src/main/java/com/vimeo/networking/RetrofitClientBuilder.java

```diff
@@ -66,6 +70,19 @@
     private List<Interceptor> networkInterceptorList = new ArrayList<>();
     private SSLSocketFactory sSLSocketFactory;
 
+    /**
+     * Add an Interceptor that will be added to all
+     * {@link OkHttpClient} that are built by this builder.
+     * This will be used as a network interceptor to
+     * intercept all network traffic.
+     *
+     * @param debugInterceptor the interceptor to use as a
+     *                         debug interceptor. Can be null.
+     */
+    public static void setDebugInterceptor(@Nullable Interceptor debugInterceptor) {
+        sDebugInterceptor = debugInterceptor;
+    }
+
```

Comment:
I think there is merit in having consumers of this library be able to add more than one other network interceptor (plus non-network interceptors). What I think is the ```Configuration``` object passed in to VimeoClient ```initialize``` can know about these, and when using the RetrofitClientBuilder, it can make use of the existing add*Interceptors.